assests/css/styles.css

```diff
@@ -97,3 +104,301 @@ h1 {
 li {
   list-style: none;
 }
+
+.section-2 {
+  width: 375px;
```

Comment:
Kindly, I suggest you not to put a static width, it is better to use a percentage so that your site adapts to any size of mobile device